src/main/java/org/codehaus/mojo/buildhelper/TimestampPropertyMojo.java

```diff
@@ -90,12 +91,32 @@
     @Parameter( defaultValue = "second" )
     private String unit;
 
+    /**
+     * The source of the time. Valid Values are
+     * <ul>
+     * <li>current</li>
+     * <li>build</li>
+     * </ul>
+     *
+     * @since 1.13
+     */
+    @Parameter( defaultValue = "current" )
+    private String timeSource;
+
     /**
      * The locale to use, for example <code>en,US</code>.
      */
     @Parameter
     private String locale;
 
+    /**
+     * The Maven Session.
+     *
+     * @since 1.13
```

Comment:
next version of the plugin would be 3.2.0

Reply:
Done